### DISCUSSION TITLE
Would reference machines to develop or file bug reports against be useful?

### DISCUSSION THREAD

**Discussion by HanHeld** [86Box/86Box#3429] [Ideas]
**Is your feature request related to a problem? Please describe.**

I am not sure this is an existing problem, but based on a conversation I had in Discord I think it may be useful to develop reference implementations for development purposes.

**Describe the solution you'd like**

The idea would be to have at least one configuration that is intended to be stable at all times for open source projects (primarily operating systems, but not exclusively) to build against. 

By "Configuration" I mean an combination of emulated components such as motherboard, cpu, video card, network card, disk controller which together are intended to represent a working computer. Does not have to be based on an existing real-world computer but that would be nice. 

I am imagining that the two features such a combination would have would be that it is "stable as in unchanging" and "stable as in guaranteed to work". 

It ought to be noted this is not to imply breakage in any other part of 86box, but acknowledge that 86box is in a state of constant development and improvement. The concept I am thinking of is similar to the idea of "LTS" release versus "rolling release" except that instead of releases I am talking about components (eg emulated computers). 

Ideally there ought to be at least three "stable" emulated computers: a PC/XT one, a 286 based one and a 386-based one. I think it would be good to also have a intended-to-be-stable pentium-based configuration as well.

**Describe alternatives you've considered**
Dev on virtualbox, play on 86box.

**Additional context**
I am writing this from the point of view of someone who finds myself writing bug reports -not from the point of view of an OS developer or the point of view of an 86Box developer. In short it might help if I can say "X does not work on 86box which is stable so that means there's a problem with your OS, and this is a bug report on that problem".

**Comment by OBattler** [comment]
For 808x and 8086, it's easy - the "stable" ones are the IBM PC's and XT's. For 286's, it's the IBM AT. For 386, there's currently no such "known to always work" machine, but we could eventually select one.

**Comment by HanHeld** [comment]
My instinct is to suggest a couple of computers (compaq, ps/2) that I realized would need inordinate amounts of configuration by the end user so...I'll have to keep thinking lol
